firefox_puppeteer/ui/menu.py

```diff
@@ -6,10 +6,10 @@
 from marionette_driver.errors import NoSuchElementException
 
 from firefox_puppeteer import DOMElement
-from firefox_puppeteer.base import BaseLib
+from firefox_puppeteer.ui_base_lib import UIBaseLib
 
 
-class MenuBar(BaseLib):
+class MenuBar(UIBaseLib):
```

Comment:
I would suggest that we continue our discussion on the appropriate bug in Bugzilla given that this PR is closed. One last comment here to answer your question. Please have a look at https://dxr.mozilla.org/mozilla-central/search?q=contentareacontextmenu&redirect=false.